src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -281,3 +281,9 @@ def prepare_logs_argument(c):
                    arg_type=get_three_state_flag(),
                    help="Disable Application Insights.",
                    validator=validate_app_insights_parameters)
+
+    with self.argument_context('spring-cloud service-registry bind') as c:
+        c.argument('app', app_name_type, help='Name of app.', validator=validate_app_name)
+
+    with self.argument_context('spring-cloud service-registry unbind') as c:
+        c.argument('app', app_name_type, help='Name of app.', validator=validate_app_name)
```

Comment:
nit: validate_app_name is used in creation for naming checking, like illegal charactor, length. I don't this it is necessary here. If service changes naming limitation in the future, this CLI version will fail to bind/un-bind app whose name is not valid in current version?